utils/lj-releng

```diff
@@ -0,0 +1,228 @@
+#!/usr/bin/env perl
```

Comment:
can we update the tool `utils/lj-releng` in a separate PR?

one PR for one thing
small PR is easier to review

Reply:
done. here is the smaller pr https://github.com/apache/apisix/pull/11862 @membphis